multiplex.go

```diff
@@ -234,6 +233,10 @@ func (mp *Multiplex) handleIncoming() {
 		mp.chLock.Lock()
 		msch, ok := mp.channels[ch]
 		mp.chLock.Unlock()
+
+		// Why the +1s? 1 is added to the tag if the message is sent by an initiator.
```

Comment:
Connection initiator versus stream initiator. We choose stream IDs based on who initiated the connection.